src/main/java/com/mysema/codegen/JavaWriter.java

```diff
@@ -481,6 +481,11 @@ public JavaWriter suppressWarnings(String type) throws IOException {
         return line("@SuppressWarnings(\"" + type + "\")");

```

Comment:
a `line(String)` call is much faster than processing the annotation, so I'd rather keep the original.